src/scripts/validators/validators.js

```diff
@@ -0,0 +1,133 @@
+// src/scripts/validators/validators.js
+export function validateAge(age) {
+  if (age == null || age === '') {
```

Comment:
```suggestion
  if (!age) {
```
You can simplify it. The same for other functions.